src/modals/getBoardsModal.ts

```diff
@@ -0,0 +1,42 @@
+import { IHttp, IModify, IPersistence, IRead, IUIKitSurfaceViewParam } from '@rocket.chat/apps-engine/definition/accessors';
+import { ModalsEnum } from '../enums/Modals';
+import { SlashCommandContext } from '@rocket.chat/apps-engine/definition/slashcommands';
+import { UIKitInteractionContext, UIKitSurfaceType } from '@rocket.chat/apps-engine/definition/uikit';
+import { Block } from '@rocket.chat/ui-kit';
+import { getActionsBlock, getButton, getInputBox, getOptions, getSectionBlock, getStaticSelectElement } from '../helpers/blockBuilder';
+
+export async function getBoardsModal({ modify, read, persistence, http, slashcommandcontext, uikitcontext, data }: { modify: IModify; read: IRead; persistence: IPersistence; http: IHttp; slashcommandcontext?: SlashCommandContext; uikitcontext?: UIKitInteractionContext; data?: string }): Promise<IUIKitSurfaceViewParam> {
```

Comment:
Same from the `api/boards/getBoards.ts`. Extract the parameters to an interface to be easier and better to maintain.

Reply:
Noted.